index.js

```diff
@@ -1,32 +1,43 @@
+const INITIAL_TIMER = 60 * 2 + 30;
 let interval,
   isPaused = false,
-  timer = 60 * 2 + 29,
+  timer = INITIAL_TIMER,
   minutes,
   seconds,
   isRunning = false,
-  display = document.querySelector("#time");
+  display = document.querySelector("#time"),
+  audio = document.querySelector("#audioBell");
 
 const images = ["./assets/Carlos.png", "./assets/Felipe.png"];
 
 window.onload = function () {
   document.getElementById("header-image").src =
     images[Math.round(Math.random() * 1)];
   hljs.initHighlightingOnLoad();
-};
 
-function startTimer(display) {
-  isRunning = true;
-  interval = setInterval(function () {
-    minutes = parseInt(timer / 60, 10);
-    seconds = parseInt(timer % 60, 10);
+  display.textContent = getMinSec(timer);
+
+};
 
-    minutes = minutes < 10 ? "0" + minutes : minutes;
-    seconds = seconds < 10 ? "0" + seconds : seconds;
+function getMinSec (timer) {
+  minutes = parseInt(timer / 60, 10);
+  seconds = parseInt(timer % 60, 10);
 
-    display.textContent = minutes + ":" + seconds;
+  minutes = minutes < 10 ? "0" + minutes : minutes;
+  seconds = seconds < 10 ? "0" + seconds : seconds;
+  
+  return minutes + ":" + seconds;
```

Comment:
```suggestion
  return `${minutes}:${seconds}`;
```

Reply:
```
const convertSecondsToTime = secs => {
--->  if (secs >= 60*60) return error <---------
  const time = new Date(secs * 1000);
  const formatZeros = obj => obj.toString().padStart(2, '0');
  const minutes = time.getUTCMinutes();
  const seconds = time.getSeconds();
  return `${formatZeros(minutes)}:${formatZeros(seconds)}`
};
```
or
```
const convertSecondsToTime = secs => {
  const minutes = parseInt(secs / 60, 10).toString().padStart(2, '0');
  const seconds = parseInt(secs % 60, 10).toString().padStart(2, '0');
  return `${minutes}:${seconds}`;
  
}
```

Reply:
I have just seen the `minutes` and `second` variables are defined at the beginning of the file. Is necessary to have them defined globally?

Reply:
I dont touch it... Actually need a full refactor, maybe next PR ;)